image.yaml

```diff
@@ -127,14 +125,18 @@ modules:
           - name: os-logging
             version: '1.0'
 packages:
-      repositories:
-          - jboss-os
-          - jboss-rhscl
+      manager: microdnf
```

Comment:
@martin-kanis The package manager can be kept to `yum`, since that's what's used in `"rhel7:7-released"` image:

```
$ docker run -it bd0240457182 which yum
/usr/bin/yum
```